pkg/operator/controller/nodeobservability/daemonset.go

```diff
@@ -235,7 +287,7 @@ func (r *NodeObservabilityReconciler) desiredDaemonSet(nodeObs *v1alpha2.NodeObs
 							VolumeSource: corev1.VolumeSource{
 								ConfigMap: &corev1.ConfigMapVolumeSource{
 									LocalObjectReference: corev1.LocalObjectReference{
-										Name: nodeObs.Name,
+										Name: "kubelet-serving-ca",
```

Comment:
Why did you drag `kubeletCAConfigMapName` object all this long way through to set the name with a constant?